docs/admin/upgrade/index.md

```diff
@@ -0,0 +1,11 @@
+# Upgrading
+
+Guidelines about upgrading CrateDB clusters.
+
+```{toctree}
+:maxdepth: 2
+
+planning
+rolling
+full
+```
```

Comment:
Also, now that we are talking about it, what do you think about improving the wording by stripping redundancy, at least on the labels within the primary navigation?

![image](https://github.com/crate/cratedb-guide/assets/453543/662f1b2e-71c4-4106-90aa-4f3fb060ebe8)

Reply:
I think "Rolling Upgrade" and "Full Restart Upgrade" are fine there.
"General Guidelines" or "Guidelines" might be sufficient though.